### ISSUE TITLE
Getting error message "Ladebereich ist voll" (Loading area is full)

### ISSUE THREAD

**Issue by Erzmaster** [loki79uk/FS25_UniversalAutoload#60]
Hi, 

i am continously getting the error message: "Der Ladebereich ist voll, es können keine weiteren Artikel dieser Größe geladen werden." (translated: Loading area is full, no more items of this size can be loaded)
![Screenshot 2024-12-02 185855](https://github.com/user-attachments/assets/3ffadbdc-b0a0-482e-9835-0b605570a580)

As you can see in the picture, my trailer is completely empty.

Kind Regards

**Comment by Erzmaster** [comment]
@loki79uk Thanks a lot for the fast response. And also for the great work - I can imagine that you put quite some hours into development! 

Understood. Is that flag saved in the savegame files somewhere and I can manually reset it? 
I made another screenshot with the developer mode on - just in case this helps you understand the issue better. Seems like the mod correctly identifies the loading areas and the items. 
![Screenshot 2024-12-02 210148](https://github.com/user-attachments/assets/2ca9c833-2660-440a-ab49-1d6af7223605)

**Comment by Erzmaster** [comment]
@loki79uk Thanks a lot for the fast response. And also for the great work - I can imagine that you put quite some hours into development! 

Understood. Is that flag saved in the savegame files somewhere and I can manually reset it? 
I made another screenshot with the developer mode on - just in case this helps you understand the issue better. Seems like the mod correctly identifies the loading areas and the items. 
![Screenshot 2024-12-02 210148](https://github.com/user-attachments/assets/2ca9c833-2660-440a-ab49-1d6af7223605)

**Comment by Erzmaster** [comment]
And also one further comment: 
You said the flag loaded is still active and it should have reset when unloaded. Well, I didnt really unload the trailer. I drove to a selling point and it sold the goods automatically (without me manually pressing unload). Could that be the cause?

**Comment by loki79uk** [comment]
I think that is the exact issue, it was never needed before and was always determined at the start of a game using the "isEmpty" function.  Maybe the best temporary solution is to store it in the savegame file for now, so that it can be reset manually as you say.  First I will see if I can clean up the usage and determine the state from the triggers alone.

Can you possibly attach the `vehicles.xml` from your savegame where it is not loading?

**Comment by loki79uk** [comment]
> And also one further comment: You said the flag loaded is still active and it should have reset when unloaded. Well, I didnt really unload the trailer. I drove to a selling point and it sold the goods automatically (without me manually pressing unload). Could that be the cause?

I'm pretty sure this is it actually!

**Comment by Erzmaster** [comment]
Sure. Attached my file [
[vehicles.zip](https://github.com/user-attachments/files/17983588/vehicles.zip)
](url)

**Comment by loki79uk** [comment]
it is worth a try while I am working on this to just remove the section from that vehicle:
```xml
        <FS25_UniversalAutoload>
            <universalAutoload>
                <currentState tipside="left" loadside="both" materialIndex="1" containerIndex="1" loadingFilter="true" baleCollectionMode="false" useHorizontalLoading="false" loadWidth="1.640472" loadHeight="0.783602" loadLength="9.680365" actualWidth="1.640472" actualLength="8.475029" layerCount="0" layerHeight="0.000000" nextLayerHeight="1.907669" loadAreaIndex="1"/>
            </universalAutoload>
        </FS25_UniversalAutoload>
```

The values are clearly wrong, suggesting it has things loaded when they are not.  It won't hurt at all to completely delete that part including all UAL tags.  The values will just be set to defaults, but it still might not work.

**Comment by Erzmaster** [comment]
Perfekt, that worked. Thanks! :)

**Comment by loki79uk** [comment]
Great, just don't drive over sell points for now.  I think it will be okay if you don't do that! :D

**Comment by Erzmaster** [comment]
Yeah, I just went to two sell points and it automatically unloaded/ sold AND it worked afterwords also. So that's that :D 
 (I was prepared to clean up my .xml afterwards, but was not needed)

**Comment by loki79uk** [comment]
hmm that is even weirder..  maybe related to saving as well?

<insert confused emoji>

**Comment by loki79uk** [comment]
I'm pretty sure that this is fixed now after 0.0.0.13 (not 10, 11 or 12..), but please re-open if you have any more issues with the trailer thinking it is full when it is not.
